modules/ROOT/pages/configuration/configuration-settings.adoc

```diff
@@ -16,6 +16,9 @@ To avoid losing changes when restarting Neo4j, make sure you update xref:configu
 
 In a clustered environment, `CALL dbms.setConfigValue` affects only the server it is run against, and it is not propagated to other members.
 If you want to change the configuration settings on all cluster members, you have to run the procedure against each of them and update their _neo4j.conf_ file.
+
+Each member of the cluster has its own _neo4j.conf_ file and thus, it is possible that the settings for the same database could be different on different members of the cluster.
```

Comment:
I'd maybe switch this up slightly and say _"Each member of the cluster has its own _neo4j.conf_ file, but it is not recommended for settings for the same database to be different across members of the cluster. As this can have unforeseen consequences."_